test/test_autograd.py

```diff
@@ -4282,6 +4282,76 @@ def manual_increase_gradient(grad):
         loss.backward()
         self.assertEqual(param.grad, 6 * param)
 
+    def test_grad_fn_attr_bindings(self):
+        # Check that the getter of each type returns what we want
+        # See `gen_autograd_functions.py` for how getters are generated
```

Comment:
Can you add a comment here that future users can safely update the tests in the future if they change any of these functions implementations.
This test is just to test the binding and not that the attributes are always present for these functions.